tests/telemetry/test_telemetry.py

```diff
@@ -153,21 +154,25 @@ def test_telemetry_enabledbydefault(duthosts, rand_one_dut_hostname):
             status_expected = "enabled";
             pytest_assert(str(v) == status_expected, "Telemetry feature is not enabled")
 
+
 def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, setup_streaming_telemetry, localhost):
     """Run pyclient from ptfdocker and show gnmi server outputself.
     """
     duthost = duthosts[rand_one_dut_hostname]
+    for asic_index in duthost.get_frontend_asic_ids():
```

Comment:
@slutati1536 please add comment: there is a point to test telemetry only if there is data port. in a case where there is no data port the test should not fail but should also not run.

Reply:
done